lazyarray.py

```diff
@@ -49,6 +54,7 @@ def wrapped_meth(self, val):
     return wrapped_meth
 
 
+
```

Comment:
please remove all these extra blank lines. There should be exactly two blank lines between top-level classes and functions, and one blank line between methods inside a class (see https://www.python.org/dev/peps/pep-0008/)